app/Controllers/Http/CommentController.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+
+const Comment = use('App/Models/Comment')
+
+class CommentController {
+  index({ request }) {
+    const { page = 1, query = '' } = request.get()
+    return Comment.query()
+      .where((q) => {
+        q.orWhere('commenter', 'ilike', `%${query}%`)
+        q.orWhere('event_id', 'ilike', `%${query}%`)
```

Comment:
I think these need to be different parameters.

`query` for comment text
`commenter_id` for `commenter_id` 
`event_id` for `event_id`

like

`?query=text&commenter=1&event_id=2`